Backend/src/routes/api.js

```diff
@@ -23,12 +24,100 @@ for (const property in DATA_FORMAT) {
 
 // Send data to front-end
 ROUTER.get("/api", (req, res) => {
-  console.time("send http");
-  const temp = res.send({ response: frontendData }).status(200);
-  temp.addListener("finish", () => console.timeEnd("send http"));
+  // console.time("send http");
+  // const temp = 
+
+  // if(graphsToSend.length){
+  // filtered graphs to send
+  let f_data = filterGraphsToSend(frontendData)
+  //console.log("SENDING GRAPH:", graphsToSend)
+  res.send({ response: f_data }).status(200);
+  // } else{
+  //   res.send({ response: {} }).status(200);
+  // }
+
+  // res.send({ response: frontendData }).status(200);
+  // temp.addListener("finish", () => console.timeEnd("send http"));
```

Comment:
```suggestion
  // filtered graphs to send
  let f_data = filterGraphsToSend(frontendData)
  
  res.send({ response: f_data }).status(200);
```
Can remove commented out lines of code